guides/vsf1_instance.md

```diff
@@ -0,0 +1,628 @@
+  `/sparta-${SUFFIX_SPLAT}`,
+  `/sprite-${SUFFIX_SPLAT}`,
+  `/stark-${SUFFIX_SPLAT}`,
+  `/stellar-${SUFFIX_SPLAT}`,
+  `/strike-${SUFFIX_SPLAT}`,
+  `/strive-${SUFFIX_SPLAT}`,
+  `/summit-${SUFFIX_SPLAT}`,
+  `/supernova-${SUFFIX_SPLAT}`,
+  `/sybil-${SUFFIX_SPLAT}`,
+  `/sylvia-${SUFFIX_SPLAT}`,
+  `/taurus-${SUFFIX_SPLAT}`,
+  `/teton-${SUFFIX_SPLAT}`,
+  `/thorpe-${SUFFIX_SPLAT}`,
+  `/tiberius-${SUFFIX_SPLAT}`,
+  `/tiffany-${SUFFIX_SPLAT}`,
+  `/torque-${SUFFIX_SPLAT}`,
+  `/tristan-${SUFFIX_SPLAT}`,
+  `/troy-${SUFFIX_SPLAT}`,
+  `/typhon-${SUFFIX_SPLAT}`,
+  `/viktor-${SUFFIX_SPLAT}`,
+  `/voyage-${SUFFIX_SPLAT}`,
+  `/vulcan-${SUFFIX_SPLAT}`,
+  `/wayfarer-${SUFFIX_SPLAT}`,
+  `/yoga-${SUFFIX_SPLAT}`,
+  `/zeppelin-${SUFFIX_SPLAT}`,
+  `/zing-${SUFFIX_SPLAT}`,
+  `/zoe-${SUFFIX_SPLAT}`,
+  `/zoltan-${SUFFIX_SPLAT}`,
+];
+// static prerendering
+router.prerender(pages.filter(page => !page.includes(SPLAT)));
+// xdn static files
+router.get('/service-worker.js', ({ serviceWorker, cache }) => {
+  cache(CACHE_ASSETS);
+  serviceWorker(`${DIST_XDN_CLIENT}/service-worker.js`);
+});
+router.get('/main.js', ({ serveStatic, cache }) => {
+  cache(CACHE_ASSETS);
+  serveStatic(`${DIST_XDN_CLIENT}/browser.js`);
+});
+// assets
+router.get(`/dist/${SPLAT}`, ({ serveStatic, cache }) => {
+  cache(CACHE_ASSETS);
+  serveStatic(`${DIST_APP}/${SPLAT}`);
+});
+router.get(`/assets/${SPLAT}`, ({ serveStatic, cache }) => {
+  cache(CACHE_ASSETS);
+  serveStatic(`${DIST_XDN_ASSETS}/${SPLAT}`);
+});
+router.get(`/img/${SPLAT}`, ({ proxy, cache }) => {
+  cache(CACHE_ASSETS);
+  proxy('origin')
+});
+// api
+router.get(`/api/catalog/${SPLAT}`, ({ proxy, cache }) => {
+  cache(CACHE_PAGES);
+  proxy('origin');
+});
+router.get(`/api/stock/${SPLAT}`, ({ proxy, cache }) => {
+  cache(CACHE_PAGES);
+  proxy('origin');
+});
+// pages
+pages.forEach(page => {
+  router.get(page, ({ cache, proxy }) => {
+    cache(CACHE_PAGES);
+    proxy(BACKENDS.js);
+  });
+});
+// fallback
+router.fallback(({ proxy }) => {
+  proxy(BACKENDS.js);
+});
+export default router;
+```
+- Add `xdn/cache.js` file. For example:
+```js
+const TIME_1H = 60 * 60;
+const TIME_4H = TIME_1H * 4;
+const TIME_1D = TIME_1H * 24;
+/**
+ * The default cache setting for pages in the shopping flow
+ */
+export const CACHE_PAGES = {
+  edge: {
+    maxAgeSeconds: TIME_4H,
+    forcePrivateCaching: true,
+    staleWhileRevalidateSeconds: TIME_1H, // this way stale items can still be prefetched
+  },
+  browser: {
+    maxAgeSeconds: TIME_4H,
+    serviceWorkerSeconds: TIME_4H,
+    spa: true,
+  },
+};
+/**
+ * The default cache setting for static assets like JS, CSS, and images.
+ */
+export const CACHE_ASSETS = {
+  edge: {
+    maxAgeSeconds: TIME_1D,
+    forcePrivateCaching: true,
+    staleWhileRevalidateSeconds: TIME_1H, // this way stale items can still be prefetched
+  },
+  browser: {
+    maxAgeSeconds: TIME_1D,
+    serviceWorkerSeconds: TIME_1D,
+    spa: true,
```

Comment:
Are you sure you need this? If so, why?

Reply:
@markbrocato I'm using that since it's an SPA app